src/Core/Components/Tabs/FluentTabs.razor.cs

```diff
@@ -264,4 +265,18 @@ private async Task DisplayMoreTabAsync(FluentTab tab)
             ActiveId = tab.Id,
         });
     }
+
+    /// <summary>
+    /// Go to a specific tab by specifying an id
+    /// </summary>
+    /// <param name="TabId">Id of the tab to goto</param>
+    /// <returns></returns>
+    public async Task GoToTab(string TabId)
```

Comment:
Because that's an `async` method, his name should be `GoToTabAsync`.

Reply:
Done